docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst

```diff
@@ -558,10 +558,14 @@ Probability of sampling from a given country's hemoglobin distribution using the
     - Unknown
     - SWEs to investigate
     - Soon
-  * - `Issues with duration of pregnancy states: [A] long duration of no maternal disorders post-pregnancy state, [B] duration of pregnancy state by pregnancy outcome does not appear to be differentiated <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/validation/maternal/model3%2C%20anemia%2C%20etc/sim_v_and_v.ipynb>`_
-    - Unknown
-    - SWEs to investigate
-    - Soon
+  * - Incident pregnancy not restricted to those in the not pregnant state
```

Comment:
Added these two to MIC-3011, and we should have fixed both. We'll get new results as soon as the cluster gets available.